templates/zerver/apps.html

```diff
@@ -26,6 +26,7 @@ <h1>Zulip for <span class="platform"></span></h1>
                         <a class="link no-action" href="">
                             <button type="button" name="button" class="green">Download Zulip for <span class="platform"></span></button>
                         </a>
+                        <p id="download-android-apk"><a href="https://github.com/zulip/zulip-mobile/releases/latest"><span style="color:white;font-size:13px">Download APK</span></a></p>
```

Comment:
I am confused here regarding the use of `<p><a><span>` structure. And styles are required to go in respective css files.